app/src/main/java/com/ubclaunchpad/room8/LoginActivity.java

```diff
@@ -24,10 +25,11 @@
 import com.ubclaunchpad.room8.Room8Utility.FirebaseEndpoint;
 import com.ubclaunchpad.room8.Room8Utility.UserStatus;
 
-public class LoginActivity extends AppCompatActivity implements View.OnClickListener {
+public class LoginActivity extends AppCompatActivity implements View.OnClickListener, View.OnTouchListener {
 
     EditText mEmail, mPassword;
     private FirebaseAuth mAuth;
+    private boolean emailEntered = false;
```

Comment:
Maybe we don't need a boolean? I think it might be enough to check that mEmail has text inside it?